src/Sylius/Bundle/InstallerBundle/Command/InstallSampleDataCommand.php

```diff
@@ -56,8 +56,8 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $doctrineConfiguration->setSQLLogger(null);
 
         $commands = array(
-            'doctrine:fixtures:load'       => array('--no-interaction' => true),
             'doctrine:phpcr:fixtures:load' => array('--no-interaction' => true),
+            'doctrine:fixtures:load'       => array('--no-interaction' => true),
```

Comment:
In product fixtures we're referencing images from Phpcr that why I had to make them load first.